apps/spruce/src/pages/task/taskTabs/TaskHistory/index.tsx

```diff
@@ -117,6 +138,30 @@ const TaskHistory: React.FC<TaskHistoryProps> = ({ task }) => {
     }
   }, [direction, setQueryParams, prevPageCursor, queryParams]);
 
+  const numMatchingResults = useMemo(
+    () =>
+      visibleTasks.reduce(
+        (acc, t) => ("isMatching" in t && t.isMatching ? acc + 1 : acc),
```

Comment:
(after change from above)
```suggestion
        (acc, t) => (t.isMatching ? acc + 1 : acc),
```